pkg/tests/common/appstudio.go

```diff
@@ -1,16 +1,20 @@
 package common
 
 import (
+	"fmt"
+	"time"
+
 	"github.com/argoproj/gitops-engine/pkg/health"
 	g "github.com/onsi/ginkgo/v2"
-	"github.com/onsi/gomega"
+	. "github.com/onsi/gomega"
 	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
 	commonCtrl "github.com/redhat-appstudio/e2e-tests/pkg/utils/common"
+	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 var (
 	// Pipelines names from https://github.com/redhat-appstudio/infra-deployments/tree/main/components/build/build-templates
-	AppStudioPipelinesNames      = []string{"analyze-devfile", "appstudio-utils"}
+	AppStudioClusterTaskNames    = []string{"analyze-devfile", "appstudio-utils"}
```

Comment:
since you list the expected names of the `ClusterTasks`, it would be also worth checking the number of all `ClusterTaks` present in the cluster - if it's really equal to `len(AppStudioClusterTaskNames)` or higher. If it's higher, then it would mean that this list of names is missing some (maybe newly added) `ClusterTask`

Reply:
@MatousJobanek I'm not sure now how to deal with it:
I see in the cluster are preacreated a lot of clusterTasks:
![image](https://user-images.githubusercontent.com/59865209/147926091-32224fa8-3456-4cb1-a232-9289a352bcfd.png)

Reply:
This could be done by using a specific label for example. Either filtering out the default ones (having `operator.tekton.dev/provider-type` label) or listing the ones created for AppStudio. 
Just looking at infra-deployments repo - it already has two more ClusterRuns than those ones listed in this array.

Reply:
@MatousJobanek I see for now the appstudio clustertask are marked with label:
```app.kubernetes.io/instance=build
```

Reply:
yeah, could be, but I'm not sure where the label is taken from... Is it added by ArgoCD or by something else? Would be worth checking it.

Reply:
Is added by ArgoCD. I will skip the check for now until we have labels defined for us

Reply:
Did you create any issue for it to track this effort? If you expect anything to be added by ArgoCD then you should create a story for them. 
Otherwise, it would be better to just filter out the other (default) ones.